### PR TITLE
Feature/containerize backend cfn

### PR DESCRIPTION
**Pull request by eternaltyro**

### CODE REVIEW

**Comment by eternaltyro** [comment]
@dakotabenjamin it's ready to be reviewed. There are a few more upgrades to be made but they are fairly minor and trivial.

**Comment by eternaltyro** [comment]
@dakotabenjamin can this be merged?

File: scripts/aws/cloudformation/tasking-manager.template.js [hunk 1]
```diff
@@ -133,296 +79,233 @@ const Parameters = {
...
-      AutoScalingRollingUpdate: {
-        PauseTime: 'PT60M',
-        MaxBatchSize: 2,
-        WaitOnResourceSignals: true
-      }
+      CapacityProviders: [
+        'FARGATE',
+        'FARGATE_SPOT'
+      ],
+      ClusterName: cf.stackName, // TODO: Change this
```
Comment (dakotabenjamin) [inline scripts/aws/cloudformation/tasking-manager.template.js:142]: Does this need to be changed?

File: scripts/aws/cloudformation/tasking-manager.template.js [hunk 2]
```diff
@@ -733,11 +647,27 @@ const Resources = {
 
 const Outputs = {
   CloudfrontDistributionID: {
-    Value: cf.ref('TaskingManagerReactCloudfront'),
+    Description: "Distribution ID for Cloudfront",
+    Value: cf.ref('FrontEndCDN'),
+    Export: {
+      // Name: cf.join('-', ['tasking-manager', cf.stackName, 'cloudfront-id']) // TODO: Remove
```
Comment (dakotabenjamin) [inline scripts/aws/cloudformation/tasking-manager.template.js:653]: I have seen a few "TODO: Remove" that can be cleaned up

**Review by dakotabenjamin — CHANGES_REQUESTED** [review]
A few small comments in the review- mostly questions about the CI workflow and some comment cleanup needed. 

I am interested in your thoughts about the steps to a deployment on staging, once we merge. It looks like we will need to do a database dump>migration from the old stack to the new one, so it would be a good practice run. What would you think about a brief playbook for that?

File: scripts/aws/cloudformation/tasking-manager.template.js (hunk 1 above)
Comment (eternaltyro) [reply scripts/aws/cloudformation/tasking-manager.template.js:142]: Not at the moment. We can do this later as it's a cosmetic change.

**Pull request closed without merging**